### ISSUE TITLE
Flickering/Flashing Framebuffer

### ISSUE THREAD

**Issue by Stef1234** [meefik/linuxdeploy#148]
Hello I've installed Debian Wheezy to my Galaxy note 10.1 (2014 ed) and ran its gui environment via the Framebuffer method. All went smooth and after editing xorg.conf it could recognize all kinds of input. 

Then I tried to do the same to a very similar device (notePro 12.2), I enabled Virtual Terminal on kernel config and predictably outputting to Framebuffer worked. Only this time my framebuffer is intermittently flashing. I tried to pause the Android stack it did not work (black screen), killed zygote and surfaceflinger services manually (after having edited init.rc from the ramdisk so that to disable the auto-restarting feature of the services), again tablet's freezing. Even tried to change resolution either through FBset or even xrandr, errors all over the place.

Last I tried was to enable the (virtual) second screen feature from "developers options". This time around I got a more rapid flashing which made the framebuffer image almost watchable (as if I am/was getting around 10-20 Hz of refresh rate). Also I get corruption all over the place from android elements, like a toast or even the pointer image.

In short nothing works as it should, and in an almost equivalent setup with equivalent kernel modification (I merely enabled "permissive" and "config_vt") it works perfectly. What gives?

Oh and here's a video showing exactly that. First slow flashing and then (after enabling the "secondary" screen) rapid flickering: https://www.youtube.com/watch?v=uUBW07HrSLo

And here's the logs from xorg and linuxdeploy as I ran the above video: http://www1.zippyshare.com/v/52020430/file.html

**Comment by meefik** [comment]
Try "Properties -> GUI settings -> Freeze Android UI -> Stop".

**Comment by erhan-** [comment]
That didn't work and Linux would't show up.

**Comment by ljones0** [comment]
I'm having a very similar problem only more serious. Managed to get over the /dev/tty0 issue but my screen flickers tpo though only for a moment and only a small part of the screen. The screen then remains completely blank. Tried freeze android option on pause and stop, but did not work.

ljones

**Comment by ljones0** [comment]
I'm having a very similar problem only more serious. Managed to get over the /dev/tty0 issue but my screen flickers too though only for a moment and only a small part of the screen. The screen then remains completely blank. Tried freeze android option on pause and stop, but did not work.

ljones
